### PR TITLE
New UI tests for Set default browser setting

### PR DESCRIPTION
**Pull request by sv-ohorvath**
Ran on Firebase 30x and passed.

### CODE REVIEW

File: app/src/androidTest/java/org/mozilla/focus/activity/SetDefaultBrowserTest.kt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+package org.mozilla.focus.activity
+
+import android.os.Build
+import org.junit.Rule
+import org.junit.Test
+import org.mozilla.focus.activity.robots.homeScreen
+import org.mozilla.focus.helpers.MainActivityIntentsTestRule
+
+/**
+ * Tests to verify the functionality of the Set default browser setting
```
Comment (AaronMT) [inline app/src/androidTest/java/org/mozilla/focus/activity/SetDefaultBrowserTest.kt:13]: nit: Default

File: app/src/androidTest/java/org/mozilla/focus/activity/robots/SettingsGeneralMenuRobot.kt [hunk 2]
```diff
@@ -5,28 +5,81 @@
...
+    fun selectAlwaysOpenWithFocus() {
+        mDevice.findObject(UiSelector().text(appName)).click()
+        mDevice.findObject(
+                UiSelector()
+                        .resourceId("android:id/button_always")
+                        .enabled(true)
+        ).click()
+    }
+
+    fun verifySwitchIsOn(checked: Boolean) {
```
Comment (AaronMT) [inline app/src/androidTest/java/org/mozilla/focus/activity/robots/SettingsGeneralMenuRobot.kt:59]: I would maybe rename to verifySwitchIsToggled

**Review by AaronMT — APPROVED** [review]

File: app/src/androidTest/java/org/mozilla/focus/activity/robots/SettingsGeneralMenuRobot.kt (hunk 2 above)
Comment (sv-ohorvath) [reply app/src/androidTest/java/org/mozilla/focus/activity/robots/SettingsGeneralMenuRobot.kt]: done

File: app/src/androidTest/java/org/mozilla/focus/activity/SetDefaultBrowserTest.kt (hunk 1 above)
Comment (sv-ohorvath) [reply app/src/androidTest/java/org/mozilla/focus/activity/SetDefaultBrowserTest.kt]: done

**Pull request merged**